# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/utils/PerformanceRecorder.h
# Commit: c07caf074089
# Full Hash: c07caf07408914b5514f89c3d46fb1dced9fef06
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-01-13 21:39:47
# Regressor Bug: 1806605
# File Overlap Count: 1
# Description:
#   Bug 1806605 - Pass TrackingId instead of nsCString to CaptureStage. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165824
# ==============================================================================

diff -r 731dcbc83284 -r c07caf074089 dom/media/utils/PerformanceRecorder.h
--- a/dom/media/utils/PerformanceRecorder.h	Fri Jan 13 14:25:00 2023 +0000
+++ b/dom/media/utils/PerformanceRecorder.h	Fri Jan 13 14:25:00 2023 +0000
@@ -151,7 +151,7 @@
     MJPEG,
   };
 
-  CaptureStage(nsCString aSource, nsCString aTrackingId, int32_t aWidth,
+  CaptureStage(nsCString aSource, TrackingId aTrackingId, int32_t aWidth,
                int32_t aHeight, ImageType aImageType)
       : mSource(std::move(aSource)),
         mTrackingId(std::move(aTrackingId)),
@@ -165,10 +165,12 @@
   }
 
   nsCString mSource;
-  nsCString mTrackingId;
+  TrackingId mTrackingId;
   int32_t mWidth;
   int32_t mHeight;
   ImageType mImageType;
+
+ private:
   mutable Maybe<nsCString> mName;
 };
 